rustic: add page (#10743)
tldr-pages/tldr: pages/common/rustic.md

# rustic

> Create fast, encrypted, deduplicated backups powered by Rust.
> More information: <https://github.com/rustic-rs/rustic>.

> - Initialize a new repository in `~/backup`:
> 
> `rustic init --repository ~/backup`
> 
> - Create a new backup of `~/important_stuff` to the repository `~/backup`:
> 
> `rustic backup --repository ~/backup ~/important_stuff`

Reviewer: ```suggestion
- Initialize a new repository:

`rustic init --repository {{/srv/rustic-repo}}`

- Create a new backup of a file/directory to a repository:

`rustic backup --repository {{/srv/rustic-repo}} {{path/to/file_or_directory}}`
```

Author: I'd change `file_or_directory` to `file_or_directory_to_backup`, would you agree here?
